docs/html/user_guide.rst

```diff
@@ -342,7 +342,7 @@ pip allows you to set all command line option defaults in a standard ini
 style config file.
 
 The names and locations of the configuration files vary slightly across
-platforms. You may have per-user, per-virtualenv or site-wide (shared amongst
+platforms. You may have per-user, per-virtualenv or global (shared amongst
```

Comment:
It may be worthwhile to mention the global config is shared also by all Python installations.